frontend/admin/layout/PageLayout.tsx

```
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Outlet, matchPath, useLocation } from 'react-router-dom';
```

ESLint

[error] 'react-router-dom' imported multiple times.

(import/no-duplicates)